inference-engine/thirdparty/movidius/XLink/shared/src/XLinkDispatcher.c

```diff
@@ -996,7 +996,13 @@ static int dispatcherClean(xLinkSchedulerState_t* curr)
 
         XLINK_RET_ERR_IF(pthread_mutex_lock(&(curr->queueMutex)) != 0, 1);
         postAndMarkEventServed(event);
-        XLINK_RET_ERR_IF(pthread_mutex_unlock(&(curr->queueMutex)) != 0, 1);
+        if (pthread_mutex_unlock(&(curr->queueMutex)) != 0) {
+            if (pthread_mutex_unlock(&clean_mutex) != 0) {
+                mvLog(MVLOG_ERROR, "Failed to unlock clean_mutex after failing to lock curr->queueMutex");
```

Comment:
Failed to unlock clean_mutex after failing to **_unlock_** curr->queueMutex

Reply:
Done